packages/app/integration-tests/tests/sandboxes.test.js

```diff
@@ -50,6 +50,7 @@ const SANDBOXES = [
   //  'gpgpu-curl-noise-yq6y8', // leva (based on stitches)
   'vx55c', // react stitches
   'fo3c0n', // vue3 with <script setup>
+  'tender-cookies-iwrwek', // console.log a `bigint`
```

Comment:
This one would be a bit more involved to set up (we'd have to store the screenshot in our CI), so I would suggest removing it for now so we can merge the PR 😄 

```suggestion
  'tender-cookies-iwrwek', // console.log a `bigint`
```